apps/astarte_pairing/lib/astarte_pairing_web/controllers/fdo_fallback_controller.ex

```diff
@@ -49,7 +49,9 @@ defmodule Astarte.PairingWeb.FDOFallbackController do
     fdo_error(conn, @cred_reuse_error)
   end
 
-  def call(conn, _error) do
+  def call(conn, error) do
+    Logger.error("error: #{inspect(error)}")
```

Comment:
error is a bit too generic and hard to track

```suggestion
    Logger.error("FDO error: #{inspect(error)}")
```